classes/DepositPointsHelper.inc.php

```diff
@@ -23,16 +23,22 @@ class DepositPointsHelper {
 	 * @param $password string
 	 * @return array|null
 	 */
-	public static function loadCollectionsFromServer($url, $username, $password) {
-		if (empty($url) OR empty($username) OR empty($password)) {
+	public static function loadCollectionsFromServer($url, $username, $password, $apikey) {
+		if (empty($url) OR (empty($username) AND empty($password)) OR empty($apikey)) {
 			return null;
```

Comment:
This is an error condition, right? I know this is old code, but we're making more use of exceptions lately, and this is a good candidate for one.